qemu/qmp.go

```diff
@@ -1400,24 +1400,29 @@ func (q *QMP) ExecHotplugMemory(ctx context.Context, qomtype, id, mempath string
 
 	defer func() {
 		if err != nil {
-			q.cfg.Logger.Errorf("Unable to hotplug memory device: %v", err)
+			q.cfg.Logger.Errorf("Unable to add memory device %s: %v", id, err)
 			err = q.executeCommand(ctx, "object-del", map[string]interface{}{"id": id}, nil)
 			if err != nil {
-				q.cfg.Logger.Warningf("Unable to clean up memory object: %v", err)
+				q.cfg.Logger.Warningf("Unable to clean up memory object %s: %v", id, err)
 			}
 		}
 	}()
 
 	args = map[string]interface{}{
-		"driver": "pc-dimm",
-		"id":     "dimm" + id,
+		"driver": driver,
```

Comment:
and now that `driver` and `id` are configurable, it would be nice to have another unit test for this function

Reply:
Done.